opentnsim/strategy.py

```diff
@@ -313,16 +318,18 @@ def get_upperbound_for_power2v(vessel, width, depth, margin=0, bounds=(0, 20)):
         result["Powerallowed_v"] = row["velocity"]
         result["P_tot"] = vessel.P_tot
         result["P_installed"] = vessel.P_installed
-        result["h_0"] = depth
-        result["z_computed"] = z_computed
-        result["h_squat"] = h_0
+        # not used but for inspection
+        result["depth"] = depth
+        result["max_sinkage"] = max_sinkage
+        result["h_0"] = h_0
 
         # update resulst dict
         results.append(result)
 
     results_df = pd.DataFrame(results)
 
     # return only dataframe up to and including the first time that P_tot == P_installed
+    # TODO: is P_tot again incl hotel power, do we want to focus on P_tot being equal to P_available_for_propulsion
```

Comment:
We don't have a P_available_for_propulsion at the moment.  So at the moment we are comparing both left and right side of the equation with hotel power.